index.html

```diff
@@ -16,12 +16,25 @@
 <body>
  <div class="container">
   <header>
+   <div class="hamburger-wrapper">
     <div class="header-photo"></div>
+     <div class="hamburger">
+      <div class="hamburger-box">
+       <div class="hamburger-inner"></div>
+      </div>
+     </div>
+    </div>
     <img class="header-logo"src="assets/images/logo.png">
```

Comment:
To nie jest jakiś tragiczny błąd, ale zamykaj tag img

    `<img class="header-logo" src="assets/images/logo.png"/>`

Reply:
Poprawiłem